characterization/robot-config.py

```diff
@@ -0,0 +1,53 @@
+{

```

Comment:
It strikes me this could be a JSON file, but I assume you have to have it in Python for the characterization system to work

Reply:
Yeah I don't really know why it was made this way. This file was actually generated by the robot-characterization application that's part of WPILib now.